java/java/src/main/java/org/whispersystems/libsignal/ecc/ECPublicKey.java

```diff
@@ -24,6 +24,13 @@ public ECPublicKey(byte[] serialized) {
     this.handle = Native.ECPublicKey_Deserialize(serialized, 0);
   }
 
+  static public ECPublicKey fromPublicKey(byte[] key) {
```

Comment:
I'd suggest putting "Bytes" in the name to mirror your accessor. (Alternately, it could be `fromDjbPublicKey` or something.)